src/standalone/Vegetation/canopy_parameterizations.jl

```diff
@@ -49,8 +51,22 @@ Returns the leaf angle distribution value for CLM G function as a function of th
 solar zenith angle and the leaf orientation index. See section 3.1 of
 https://www2.cesm.ucar.edu/models/cesm2/land/CLM50_Tech_Note.pdf
 """
-function compute_G(G::CLMGFunction{FT}, θs::FT) where {FT}
-    ϕ1 = 0.5 - 0.633 * G.χl - 0.33 * G.χl^2
+function compute_G(G::CLMGFunction, θs::FT) where {FT}
+    return compute_G_CLMG.(G.χl, θs)
+end
+
+"""
+    compute_G_CLMG(
+        χl::FT,
+        θs::FT,
+    )
+
+Returns the leaf angle distribution value for CLM G function as a function of the
```

Comment:
I think it is helpful to note that this function applies pointwise (it is broadcasted over fields)